template/scripts/post-install.sh

```diff
@@ -1,15 +1,13 @@
 #1/use/bin/env bash
 
-yarn jetify
-
```

Comment:
Do we have any cmd for `jetify`?

Reply:
The file `gradle.properties` contains this feature `android.enableJetifier=true`, so we can delete it